README.md

```diff
@@ -10,10 +12,30 @@
 
 4. Are there unexpected regions with higher stars (rare patterns)? Something non-trivial that may be discovered are "hidden gems" such as food cart street or local spots
 
+5. Are more active yelpers prone to giving higher ratings or more thorough reviews?
+
+6. Do some yelpers give generally negative/positive reviews?
+
+7. Are there common words present in reviews with similar ratings?
+
+8. How consistent are ratings with common keywords?
+
+9. Do business ratings change over time? Can this change be attributed to attributes in the review data?
```

Comment:
I like this one as well. But might be computationally expensive to track a rolling average for each business

Reply:
I really like this question because you can really vary how time is handled. For example, looking at the change in ratings over days or weeks may take a lot of power. But, if we just want to get a more macroscopic view of what may be happening to a business (new or old), then you could take the average over months or subsets of months (seasons). 

In the end though, we may want to set this aside to focus more on text reviews since that seems to be the root of many of our questions.

Reply:
Oh great point! I hadn't considered slicing time in larger chunks. Adding time as a dimension would really be fascinating.  

And to @paco2633's follow up question we may be able to track something to the effect of: 
-  "Top 10 restaurants with the largest change in average review score"

___

Just want to make sure, am I thinking about this right? Not that we have to do this, but we can slice reviews with **_up to_** 3-Dimensions. Like if some patterns are sensitive to: **location**, **time**, or the **business category**.

Q: What are frequent features of 1-star (negative) and 5-star (positive) text reviews for **different categories** of business? Are there nuanced patterns amongst **major cities** in the US? And how are ratings affected **by seasons**?

- Frequently used words
- Review length
- User's review count
- User's friend network
- User's average star rating

Reply:
I think the three dimensions could make our goals more succinct. In fact, "different categories of business" may be too broad? Perhaps we should just explicitly select "different categories of restaurants". This would probably help our data reduction, too.

Reply:
Yeah that sounds perfect, this'll cut down our search space a lot

Reply:
Hopefully, because the data is well structured, it becomes fairly easy to build many different combinations for comparison. It is probably best to throw a couple defined attribute sets in the proposal so we have some definite goals to show progress. In the same line of thinking....reducing the total set using time/category/etc is probably good to start out with as a concrete goal since it is likely easier to work with (cpu timewise)